src/plugins/timelist/TimelistComponent.vue

```diff
@@ -235,17 +297,30 @@ export default {
         sourceMap: this.domainObject.sourceMap
       });
     },
+    async getActivityStates() {
+      this.activityStatesObject = await this.openmct.objects.get('activity-states');
```

Comment:
this can just be a local variable since it's never used again. should we also handle the case where the object is not found?

Reply:
There is a missing object interceptor that will create this object if it's not found.

Reply:
OK. We should still make activityStatesObject a local variable since it's only used in this block